packages/client/src/store/services/authService.ts

```diff
@@ -43,7 +43,24 @@ export const authAPI = createApi({
       }),
     }),
     getUserInfo: builder.query<UserInfoDTO, unknown>({
-      query: () => API.GET_USER_INFO,
+      query: () => {
```

Comment:
От return можно избавиться:
```suggestion
        query: () => ({
        url: API.GET_USER_INFO,
        responseHandler: response => {
          if (response.status === 200) {
            return response.json().then(json => {
              localStorage.setItem(LOCAL_STORAGE_CONSTANTS.USER_ID, json.id)
              localStorage.setItem(LOCAL_STORAGE_CONSTANTS.USENAME, json.login)
              return json
            })
          }
          return Promise.resolve(response)
        },
      }),
```